test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua

```diff
@@ -0,0 +1,86 @@
+ -----  Name of requirement that is covered-----
+ ----- Certificate have EMPTY in pt_preloaded JSON of module_config section
+ ----- Verification criteria------
+ -----  SDL must continue working as assigned.
+
```

Comment:
@Aleksandriys, please you in script the format of header from our approved script template.